account-test/specs/step-definitions/signin.steps.js

```diff
@@ -0,0 +1,79 @@
+//const {  defineFeature, loadFeature} = require('jest-cucumber')
+import {  defineFeature, loadFeature} from 'jest-cucumber'
+import SignInPage from '../../src/pages/signIn.page.js'
+const assert = require('assert');
+const feature = loadFeature('./specs/features/login.feature');
+const { Builder, By, Key, until } = require('selenium-webdriver')
+const { querySelector } = require('../../src/util/helpers.js')
+
+defineFeature(feature, test => {
+    let driver
+    let signInPage
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 1000 * 60 * 5
+    
+    beforeEach(async () => {
```

Comment:
Can the driver setup and teardown be a beforeAll so we only set this up once rather than for every test?